blueocean-dashboard/src/main/less/core.less

```diff
@@ -408,3 +408,17 @@ table.highlight {
     }
 }
 
+// FIXME This is probably still not ideal. Belongs in JDL, also.
```

Comment:
Some guy was nagging me about getting it done ASAP, and I don't know what other changes might be in JDL master at the moment. We need a proper fix for this problem, rather than promoting kludges.